mautic/mautic
# Phpstan level 1

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Automated tests included? | Y
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | /
| BC breaks? | Y
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Bunping PHPSTAN to level 1. It contains some BC changes and so I'd like to get it in 3.0.0-beta. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Hard to tell. But PHPSTAN discovered some nasty bugs.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Click through the UI. There should be no errors.

#### List backwards compatibility breaks:
1. Check the Upgrade doc changes in this PR
